tracekit.js

```diff
@@ -233,11 +233,10 @@ TraceKit.report = (function reportModuleWrapper() {
      */
     function processLastException() {
         var _lastExceptionStack = lastExceptionStack,
-            _lastArgs = lastArgs;
-        lastArgs = null;
+            _lastException = lastException;
```

Comment:
Do we need to keep track of if this is a window error or not? I don't think we do but I do know the last args contained that.